### PR TITLE
Docker updates

### PR DESCRIPTION
**Pull request by jfilak**

### CODE REVIEW

File: src/daemon/abrt-action-save-container-data.c [hunk 1]
```diff
@@ -29,58 +29,109 @@ void dump_docker_info(struct dump_dir *dd, const char *root_dir)
...
+
+        last = strrchr(last, '-');
+        if (last == NULL)
+        {
+            log_debug("The docker mount point has unknown format");
+            continue;
+        }
+
+        ++last;
+        container_id = xstrndup(last, 12);
```
Comment (mhabrnal) [inline src/daemon/abrt-action-save-container-data.c]: Could you please use some constant instead of integer 12?

Comment (jfilak) [inline src/daemon/abrt-action-save-container-data.c]: Sorry, I don't see any reason for it.  The value is literally a magic constant. I will add a comment.

**Comment by jfilak** [comment]
I've added the comment describing the magic value 12.

**Comment by mhabrnal** [comment]
Looks good to me.

**Pull request merged**